R/connect_inbo_dbase.R

```diff
@@ -46,15 +46,12 @@ connect_inbo_dbase <- function(database_name, autoconvert_utf8 = TRUE) {
         type <- "INBO PRD Server"
     }
 
-    sql_driver <- if (.Platform$OS.type == "unix") {
```

Comment:
you've lost the critical `sql_driver <-` part here

Reply:
Woops...